echidna/scripts/dump_shrink.py

```diff
@@ -0,0 +1,100 @@
+""" Example shrinking script
+
+This script:
+  * Reads in spectrum from hdf5
+  * shrinks spectrum with user input pars and low high values
+  * Saves shrunk spectrum to given output or, by default, to the same
+    directory with _shrink added to the filename.
```

Comment:
Sorry to be pedantic, but could you stick with the convention of capitalising the start of each bullet point and only ending with a full-stop if it spans multiple sentences.